docs/page/lsp-org.md

```diff
@@ -0,0 +1,41 @@
+# Literate programming using LSP and org-mode(alpha)
+`lsp-mode` provides **experimental** support for running the language servers
+inside of [org-mode](https://orgmode.org/) source blocks. `lsp-mode` is doing
+that by obtaining the information about the source block and then translating
+the point to the LSP positions back and forth so the server is which so the
+server actually thinks that Emacs has opened the original file. In order that to
+work the files has to be present on the disk as well because the server expects
+to find them.
```

Comment:
```suggestion
`lsp-mode` provides **experimental** support for running the language servers inside of [org-mode](https://orgmode.org/) source blocks. `lsp-mode` is doing that by obtaining the information about the source block and then translating the point to the LSP positions back and forth so the server is which so the server actually thinks that Emacs has opened the original file. In order to that work the files have to be present on the disk as well because the server expects to find them.
```